src/components/componentContainer.ts

```diff
@@ -6,54 +6,36 @@ module Plottable {
    * It will not do anything if instantiated directly.
    */
   export class ComponentContainer extends Component {
-    private _components: Component[] = [];
```

Comment:
Why is this being removed? Aren't classes that extend ComponentContainer all based on some base Component[]?